packages/nuxt/src/module.ts

```diff
@@ -106,10 +109,10 @@ export default defineNuxtModule<ModuleOptions>({
     addTypeTemplate(
       {
         filename: 'types/clerk.d.ts',
-        getContents: () => `import type { SignedInAuthObject, SignedOutAuthObject } from '@clerk/backend/internal';
+        getContents: () => `import type { SessionAuthObject } from '@clerk/backend';
           declare module 'h3' {
-            type AuthObjectHandler = (SignedInAuthObject | SignedOutAuthObject) & {
-              (): SignedInAuthObject | SignedOutAuthObject;
+            type AuthObjectHandler = SessionAuthObject & {
+              (): SessionAuthObject;
```

Comment:
This is a new type introduced in machine auth

see https://github.com/clerk/javascript/blob/6e9bde86e1aa50ca07cb4617647f27ad12660649/packages/backend/src/tokens/types.ts#L180,L83